HRIS.Services.Tests/Services/EmployeeSalaryDetailsServiceUnitTest.cs

```diff
@@ -0,0 +1,169 @@
+﻿//TODO: Complete Unit Tests
+
```

Comment:
Absolutely not, remove this file and put the tests in a different PR

Reply:
Apologies, I removed the unit tests and added them to a separate branch